modules/ROOT/pages/troubleshoot-api-provisioning-issues.adoc

```diff
@@ -53,19 +53,14 @@ To resolve this issue:
 . Resolve the username and password error.
 . Re-create the API.  
 
+[[api-spec-file-size-exceeded]]
+== API Specification File Exceeds Supported Size
 
-== API Spec size exceeds allowed supported size
-
-When provisioning an API with a large specification file (current limit is `550KB`) you might face some issues on the provision operation or even the Service Mesh services.
+When provisioning an API with a specification file that is larger than 550 KBs, you might encounter issues with the provision operation or the Service Mesh services.
 
 === Cause
 
-If you use a large file, the following errors might show when listing `service-mesh` namespace's pods:
-
-image::large-api-error-oom-pod.png[Pod restarted due to out of memory error]
-image::large-api-error-crashloop-backoff-pod.png[Pod restarted due to crashloop backoff error]
-
-Or when inspecting the Service Catalog logs, an error of this kind:
+If you use a large file, the pods might be restarted due to insufficient memory or crashloop errors:
```

Comment:
Neither of this errors (`insufficient memory or crashloop errors`) correspond to the following code block, that block shows Service Catalog limitation errors. 
The insufficient memory or crashloop errors are shown in the images uploaded.